SPECS/qt5-qtsvg/CVE-2023-32762.nopatch

```diff
@@ -0,0 +1,2 @@
+The patch for the CVE-2023-32762 in qt5-qtbase.spec itself fixes this CVE for qt5-qtsvg as well.
+It seems our tools are detecting this in qt5-qstsvg because it has dependency on qt5-qtbase-devel.
```

Comment:
```suggestion
It seems our tools are detecting this in qt5-qtsvg because it has dependency on qt5-qtbase-devel.
```

Reply:
thanks @mfrw for the approval. I have been suggested to raise a dispute for clearing this CVE instead of this and hence closing the PR for now.